tests/spec/fs.read.spec.js

```diff
@@ -33,10 +34,10 @@ describe('fs.read', function() {
   });
 
   it('should update the current file position', function(done) {
-    var fs = util.fs();
-    var wbuffer = Buffer.from([1, 2, 3, 4, 5, 6, 7, 8]);
-    var rbuffer = Buffer.alloc(wbuffer.length);
-    var _result = 0;
+    const fs = util.fs();
+    const wbuffer = Buffer.from([1, 2, 3, 4, 5, 6, 7, 8]);
+    const rbuffer = Buffer.alloc(wbuffer.length);
+    const _result = 0;
```

Comment:
This needs to be `let`, since it gets changed in lines 52 and 55 below.  This is why travis is failing the build:

```
/home/travis/build/filerjs/filer/tests/spec/fs.read.spec.js
  52:11  error  '_result' is constant  no-const-assign
  55:13  error  '_result' is constant  no-const-assign
```